src/Compilers/CSharp/Test/Emit2/Semantics/ExperimentalAttributeTests.cs

```diff
@@ -77,18 +80,779 @@ public class C
         var src = """
 C.M();
 """;
+        var comp = CreateCompilation(new[] { src, libSrc, experimentalAttributeSrc });
+        comp.VerifyDiagnostics();
```

Comment:
>VerifyDiagnostics() [](http://example.com/codeflow?start=13&length=19)

Should there be an error reported for `C.M()`?

Reply:
> Are we not reporting errors with experimental APIs within an assembly with an assembly-level [Experimental] attribute?

Correct. That's the second part of the change (described in OP)